packages/server/graphql/public/types/User.ts

```diff
@@ -57,6 +64,53 @@ const User: UserResolvers = {
     const stripeLineItems = await fetchAllLines(invoiceId)
     const invoice = await manager.retrieveInvoice(invoiceId)
     return generateInvoice(invoice, stripeLineItems, orgId, invoiceId, dataLoader)
+  },
+  availableTemplates: async ({id: userId}, {first, after}, {authToken, dataLoader}) => {
```

Comment:
+1 is it possible to combine type definition via SDL and the old way? if yes, we should try to use the new way when we can

Reply:
Could you elaborate? This is on the new SDL `User` type.

Reply:
oh god, long day, sorry 🤦